src/compiler/types.ts

```diff
@@ -376,18 +376,12 @@ namespace ts {
         // If this node was parsed in the 'yield' context created when parsing a generator.
         Yield = 1 << 2,
 
-        // If this node was parsed in the parameters of a generator.
-        GeneratorParameter = 1 << 3,
-
         // If this node was parsed as part of a decorator
         Decorator = 1 << 4,

```

Comment:
I was going to do that once the change to remove 'strict mode' gets merged into this.